integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -195,7 +192,6 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
     sql("USE default")
     sql("DROP DATABASE IF EXISTS streaming CASCADE")
     new File("target/csvdatanew").delete()
```

Comment:
it should be integrationPath + "/spark2/target/csvdatanew

Reply:
fixed